src/Illuminate/Console/Scheduling/ScheduleListCommand.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Illuminate\Console\Scheduling;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Helper\Table;
+
+class ScheduleListCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'schedule:list';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'List the scheduled tasks';
+
+    /**
+     * The schedule instance.
+     *
+     * @var \Illuminate\Console\Scheduling\Schedule
+     */
+    protected $schedule;
+
+    /**
+     * The table headers for the command.
+     *
+     * @var array
+     */
+    protected $headers = ['Command', 'Expression', 'Maintenance', 'Overlap', 'Output', 'Append'];
+
+    /**
+     * Create a new command instance.
+     *
+     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
+     * @return void
+     */
+    public function __construct(Schedule $schedule)
+    {
+        $this->schedule = $schedule;
+
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function fire()
+    {
+        $events = $this->getEvents();
+
+        if (empty($events)) {
+            return $this->info('Your application has no scheduled commands to run.');
```

Comment:
Maybe different message? "There are no scheduled commands to list" or something.

The error message sounds like it's trying to run them (which it is not)